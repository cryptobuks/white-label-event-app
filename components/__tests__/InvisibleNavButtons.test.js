import React from 'react';
import renderer from 'react-test-renderer';
import { InvisibleNavButtons } from '../';

describe('InvisibleNavButtons', () => {
  it('renders correctly', () => {
    const comp = <InvisibleNavButtons index={0} total={5} onNextTap={() => {}} />;

    const tree = renderer.create(comp).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
