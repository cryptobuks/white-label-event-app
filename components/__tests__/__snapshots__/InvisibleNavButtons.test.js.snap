// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InvisibleNavButtons renders correctly 1`] = `
<View
  style={
    Array [
      undefined,
      Object {
        "backgroundColor": "transparent",
        "left": 154,
        "position": "absolute",
        "top": 56,
      },
    ]
  }
>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    hitSlop={undefined}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "height": 48,
        "width": 144,
      }
    }
    testID={undefined}
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": "transparent",
            "flex": 1,
          },
          Object {
            "height": 48,
            "width": 144,
          },
        ]
      }
    />
  </View>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    hitSlop={undefined}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Array [
        Object {
          "left": 214,
        },
        Object {
          "height": 48,
          "width": 144,
        },
      ]
    }
    testID={undefined}
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": "transparent",
            "flex": 1,
          },
          Object {
            "height": 48,
            "width": 144,
          },
        ]
      }
    />
  </View>
</View>
`;
